code/modules/uplink/uplink_items.dm

```diff
@@ -1881,6 +1881,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 3
 	manufacturer = /datum/corporation/traitor/donkco
 	restricted_roles = list("Roboticist", "Research Director")
+	
+/datum/uplink_item/role_restricted/clowngrenade
+	name = "C.L.U.W.N.E. Grenade"
+	desc = "A tactical clown grenade which conjures a clown from the netherworlds. Be careful, it may attack you too! Not guaranteed to be useful in the slightest."
```

Comment:
```suggestion
	desc = "A tactical clown grenade which conjures a clown from the netherworlds. Be careful; it may attack you too! Not guaranteed to be useful in the slightest."
```